### PR TITLE
Update jenkins to 1.462.2

### PR DESCRIPTION
**Pull request by notmyown**

### CODE REVIEW

**Comment by notmyown** [comment]
Depending on: https://github.com/cloudogu/dogu-build-lib/pull/50 to get into prod

File: Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@
 FROM registry.cloudogu.com/official/java:17.0.12-1
 
 LABEL NAME="official/jenkins" \
-      VERSION="2.452.4-2" \
+      VERSION="2.462.2-1" \
```
Comment (nfranzeck) [inline Dockerfile:5]: This part of the Dockerfile will be updated as part of the release process. No actions needed before release

File: dogu.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "Name": "official/jenkins",
-  "Version": "2.452.4-2",
+  "Version": "2.462.2-1",
```
Comment (nfranzeck) [inline dogu.json:3]: see above

**Review by nfranzeck — COMMENTED** [review]
Versions in Docker label and dogu.json don't need to be updated during feature development. It is part of the release process.

**Review by nfranzeck — APPROVED** [review]

**Pull request merged**